test/test.py

```diff
@@ -5,20 +5,7 @@
 import torchaudio
 import math
 import os
-
-
-class AudioBackendScope:
-    def __init__(self, backend):
-        self.new_backend = backend
-        self.previous_backend = torchaudio.get_audio_backend()
-
-    def __enter__(self):
-        torchaudio.set_audio_backend(self.new_backend)
-        return self.new_backend
-
-    def __exit__(self, type, value, traceback):
-        backend = self.previous_backend
-        torchaudio.set_audio_backend(backend)
+from _test import AudioBackendScope, BACKENDS, BACKENDS_MP3
```

Comment:
The name of the module, `_test`, is not very descriptive.
Also, to me, prefixing with underscore does not give additional context, as test modules are never intended for external use. So, something like `backend_utils` sound more appropriate.

Reply:
Good point. In fact, I just moved those functions in `common_utils.py` :)